src/AppBundle/Controller/ContactController.php

```diff
@@ -71,7 +71,7 @@ public function getContactsAction()
             ['id' => 3, 'firstname' => 'AquaPelham', 'lastname' => 'leanna', 'phonenumber' => '7890123', 'bday' => 'Aug. 20 2015', 'address' => 'tallinn'],
     ];
         $data = [
-            'contact' => $contact
+            'contacts' => $contact
```

Comment:
And once I got the other spot fixed, I noticed that we had a simple mis-match between our JSON endpoint and our JavaScript (which expects the key to be `contacts`).